### PR TITLE
Clarify, update usage of stops, calendar_attributes, route_patterns fields

### PR DESCRIPTION
**Pull request by jfabi**
**Please note that there is no actual change in the MBTA GTFS file associated with this pull request, merely updates to the documentation.** The changes referenced for the [calendar_attributes.txt](https://github.com/mbta/gtfs-documentation/blob/master/reference/gtfs.md#calendar_attributestxt) and [route_patterns.txt](https://github.com/mbta/gtfs-documentation/blob/master/reference/gtfs.md#route_patternstxt) have already taken effect in previous months.

- In [calendar_attributes.txt](https://github.com/mbta/gtfs-documentation/blob/master/reference/gtfs.md#calendar_attributestxt), updates to note that `rating_description` is included for all services, including on Ferry.
- In [route_patterns.txt](https://github.com/mbta/gtfs-documentation/blob/master/reference/gtfs.md#route_patternstxt), updates possible forms of `route_pattern_name` and removes the constraint on Commuter Rail routes having limited route patterns.
- In [stops.txt](https://github.com/mbta/gtfs-documentation/blob/master/reference/gtfs.md#stopstxt), clarifies expected values for `on_street` and `at_street`.

### CODE REVIEW

File: reference/gtfs.md [hunk 1]
```diff
@@ -313,8 +311,8 @@ parent_station | Optional | Included (some records) | For stops location within
...
 municipality | Experimental | Included (some records) | Lists the name of the city or town in which the stop is located.
-on_street | Experimental | Included (some records) | Provides the full name of the street along which a stop is located. Populated for most, but not all, bus stops. Not currently populated for rail or ferry stops, nor for stops with `location_type` not equal to `0`.
-at_street | Experimental | Included (some records) | Provides the full name of a stop's nearest cross street. Populated for most, but not all, bus stops. Note that `at_street` will not be populated unless `on_street` is also populated.
+on_street | Experimental | Included (some records) | Provides the full name of the street along which a stop is located. Newer or recently renamed or relocated streets may not properly appear in `on_street`. Populated for most, but not all, bus stops. Populated for most, but not all, bus stops. Not currently populated for rail or ferry stops, nor for stops with `location_type` not equal to `0`.
```
Comment (bklebe) [inline reference/gtfs.md:314]:
```suggestion
on_street | Experimental | Included (some records) | Provides the full name of the street along which a stop is located. Newer or recently renamed or relocated streets may not properly appear in `on_street`. Populated for most, but not all, bus stops. Not currently populated for rail or ferry stops, nor for stops with `location_type` not equal to `0`.
```

File: reference/gtfs.md [hunk 2]
```diff
@@ -313,8 +311,8 @@ parent_station | Optional | Included (some records) | For stops location within
...
+at_street | Experimental | Included (some records) | Provides the full name of a stop's nearest cross street. The nearest cross street can be on the same or opposite side of the street as the stop, and thus will not always the same street as the one referenced in `stop_name`. Newer or recently renamed or relocated streets may not properly appear in `at_street`. Populated for most, but not all, bus stops. Note that `at_street` will not be populated unless `on_street` is also populated.
```
Comment (bklebe) [inline reference/gtfs.md:315]:
```suggestion
at_street | Experimental | Included (some records) | Provides the full name of a stop's nearest cross street. The nearest cross street can be on the same or opposite side of the street as the stop, and thus will not always be the same street as the one referenced in `stop_name`. Newer or recently renamed or relocated streets may not properly appear in `at_street`. Populated for most, but not all, bus stops. Note that `at_street` will not be populated unless `on_street` is also populated.
```

**Review by bklebe — CHANGES_REQUESTED** [review]
Just a couple small typos. Did you mean to say at_street and on_street in your description?

File: reference/gtfs.md (hunk 2 above)
Comment (jfabi) [reply reference/gtfs.md]: All good catches, thanks! Ready for another look.

**Review by bklebe — APPROVED** [review]
Thanks!

**Pull request merged**